src/ast/mod.rs

```diff
@@ -2795,6 +2817,63 @@ impl fmt::Display for Statement {
                 }
                 Ok(())
             }
+            Statement::CopyIntoSnowflake {
+                into,
+                from_stage,
+                from_stage_alias,
+                stage_params: _stage_params,
```

Comment:
Why are stage_params ignored?

Reply:
Seems like `stage_params` were totally forgotten for `copy into`. Thanks for spotting this. 
I've included `stage_params` support in recent commit version.